glusterd2/commands/volumes/volfiles.go

```diff
@@ -1,10 +1,13 @@
 package volumecommands
 
 import (
+	"fmt"
 	"net/http"
 
 	restutils "github.com/gluster/glusterd2/glusterd2/servers/rest/utils"
 	volgen "github.com/gluster/glusterd2/glusterd2/volgen2"
+	"github.com/gluster/glusterd2/pkg/errors"
+	"github.com/gorilla/mux"
```

Comment:
We follow a convention for grouping imports. See **Imports:** section from this document: https://github.com/gluster/glusterd2/blob/master/doc/coding.md